temporalio/converter.py

```diff
@@ -1084,6 +1085,29 @@ async def encode_wrapper(
         """
         return temporalio.api.common.v1.Payloads(payloads=(await self.encode(values)))
 
+    async def _encode_user_metadata(
```

Comment:
Since this and the decode equivalent are only called from the client file, can they be made as just underscore-prefixed module-level helpers at the bottom of the client file?